### DISCUSSION TITLE
GraphQL mutations for Gists?

### DISCUSSION THREAD

**Discussion by Sivanesh-S** [community/community#24552] [Apps, API and Webhooks]
<p>I used v3 for all gist based operations and it worked fine. Due to over fetching and under fetching drawbacks, I decided to upgrade to v4.</p>
<p><strong>I’m new to GraphQL</strong>. I found ways to <strong>GET</strong> gist data from the query. Thanks for the perfect docs. But I couldn’t find a way to mutate (update/delete) gist’s data using Github’s v4 API.</p>
<p>My doubt is, has it not yet implemented? or It is implemented and I don’t know how to get the docs properly.</p>
<p>Thanks in Advance.</p>

**Comment by ilia-cy** [comment, marked as answer]
<p>I don’t think it is currently implemented.</p>
<p>As far as the documentation goes, these are the current supported GraphQL mutations: <a href="https://developer.github.com/v4/mutation/" rel="nofollow noopener">https://developer.github.com/v4/mutation/</a></p>
